bundles/org.openhab.binding.vesync/README.md

```diff
@@ -208,6 +208,21 @@ Number:Dimensionless LoungeAHTarget            "Lounge Air Humidifier Target Hum
 Number:Dimensionless LoungeAHMistLevel         "Lounge Air Humidifier Mist Level"                             { channel="vesync:airHumidifier:vesyncServers:loungeHumidifier:mistLevel" }
 ```
 
+#### Air Humidifier Oasis Mist Smart Model
+
+```
```

Comment:
```suggestion
```java
```

Reply:
@jlaur - Thank you. I'm surprised by this one as, my IDE def, does not like non java code having that marker. Like before on searching I found it in many other binding's- so have aligned it to the convention. Thanks again.